### PR TITLE
docs: Add LICENSE and update package.json files

### PR DESCRIPTION
**Pull request by pmareke**
Complete package information

### CODE REVIEW

**Review by LuisMartinez15 — DISMISSED** [review]

**Review by LuisMartinez15 — APPROVED** [review]

**Review by LuisMartinez15 — CHANGES_REQUESTED** [review]
I totally forgot yesterday about this, but, I think we need to create a `CHANGELOG.md` file with the current `README.md` content. We have put something like this in the `README.MD` for now as a placeholder:

# x-get-safe-property-chain

A utility for safely retrieving nested values.

### Installation

You can install the package using `npm`

```
npm install @empathyco/x-get-safe-property-chain --save-dev
```

### Usage

```
import { getSafePropertyChain } from '@empathyco/x-get-safe-property-chain';

const obj = { nestedObject: { anotherNestedObject: { message: 'Hell yeah!' } } }; 
const message = getSafePropertyChain(obj, 'nestedObject.anotherNestedObject.message');
console.log(message); // 'Hell yeah!'
```

I think we should do this because now `lerna` is going to control the package versions and it will add the changes to the `CHANGELOG.md` (as we are doing in `x-components` with `standard-version`).

File: packages/get-safe-property-chain/package.json [hunk 1]
```diff
@@ -16,10 +16,12 @@
     "type": "git",
     "url": "git+https://cbn-javieri@bitbucket.org/colbenson/get-safe-property-chain.git"
```
Comment (LuisMartinez15) [inline packages/get-safe-property-chain/package.json:17]:
```suggestion
    "url": "git+https://github.com/empathyco/x.git"
```

**Review by LuisMartinez15 — CHANGES_REQUESTED** [review]

**Review by LuisMartinez15 — APPROVED** [review]

File: packages/get-safe-property-chain/package.json [hunk 2]
```diff
@@ -14,12 +14,14 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://cbn-javieri@bitbucket.org/colbenson/get-safe-property-chain.git"
+    "url": "git+https://github.com/empathyco/x.git"
```
Comment (LuisMartinez15) [inline packages/get-safe-property-chain/package.json:17]: This should be the url to the repo, without `git+ssh` or `git+https` as specify in the [documentation examples](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#repository).

**Review by LuisMartinez15 — CHANGES_REQUESTED** [review]

File: packages/get-safe-property-chain/package.json (hunk 2 above)
Comment (pmareke) [reply packages/get-safe-property-chain/package.json:17]: I'll fix this `repository` property in the last PR.

**Review by herrardo — APPROVED** [review]

**Review by LuisMartinez15 — APPROVED** [review]

**Pull request merged**